### PR TITLE
feat(db): Add dates related to builds in the AppStore Connect table

### PR DESCRIPTION
**Pull request by relaxolotl**
Added two additional dates to provide more context to users about
the current state of their AppStore Connect integration.

### CODE REVIEW

**Review by flub — APPROVED** [review]

**Comment by wedamija** [comment]
@relaxolotl Please don't merge database changes without approval from @getsentry/owners-migrations

**Pull request merged**
